lib/segment/src/segment.rs

```diff
@@ -506,7 +506,8 @@ impl Segment {
                 // Without this check, the service will panic on the `get_vector` call.
                 Err(OperationError::InconsistentStorage {
                     description: format!(
-                        "Vector storage is inconsistent, total_vector_count: {}, point_offset: {}",
+                        "Vector storage '{}' is inconsistent, total_vector_count: {}, point_offset: {}",
+                        vector_name,
                         vector_storage.total_vector_count(),
                         point_offset
```

Comment:
I have a slight preference for this style, feel free to choose either & merge.

```suggestion
                        "Vector storage '{vector_name}' is inconsistent, total_vector_count: {}, point_offset: {point_offset}",
                        vector_storage.total_vector_count(),
```

Thanks!